api/client/service/update.go

```diff
@@ -370,6 +380,29 @@ func updateMounts(flags *pflag.FlagSet, mounts *[]mounttypes.Mount) {
 	*mounts = newMounts
 }
 
+func updateGroups(flags *pflag.FlagSet, groups *[]string) error {
```

Comment:
I guess it's ok, just looks a little strange. I suppose either we take cSpec (pointer) or a pointer to cSpec.Groups.